scripts/exgfs_wave_post_pnt.sh

```diff
@@ -146,116 +120,82 @@
     fi
   fi
 
-  if [ -s buoy.loc ]
+  if [[ -s buoy.loc ]]
   then
-    set +x
     echo "   buoy.loc and buoy.ibp copied and processed (${PARMgfs}/wave/wave_${NET}.buoys)."
-    set_trace
   else
-    set +x
-    echo ' '
-    echo '************************************* '
-    echo ' FATAL ERROR : NO BUOY LOCATION FILE  '
-    echo '************************************* '
-    echo ' '
-    set_trace
-    err=3; export err;${errchk}
-    exit $err
-    DOSPC_WAV='NO'
-    DOBLL_WAV='NO'
+    export err=3
+    err_chk 'FATAL ERROR: NO BUOY LOCATION FILE'
   fi
 
 # 1.c Input template files
 
-  if [ -f ${PARMgfs}/wave/ww3_outp_spec.inp.tmpl ]
+  if [[ -f "${PARMgfs}/wave/ww3_outp_spec.inp.tmpl" ]]
   then
-    cp -f ${PARMgfs}/wave/ww3_outp_spec.inp.tmpl ww3_outp_spec.inp.tmpl
+    cp -f "${PARMgfs}/wave/ww3_outp_spec.inp.tmpl" ww3_outp_spec.inp.tmpl
   fi
 
-  if [ -f ww3_outp_spec.inp.tmpl ]
+  if [[ -f ww3_outp_spec.inp.tmpl ]]
   then
-    set +x
     echo "   ww3_outp_spec.inp.tmpl copied. Syncing to all grids ..."
-    set_trace
   else
-    set +x
-    echo ' '
-    echo '*********************************************** '
-    echo '*** ERROR : NO TEMPLATE FOR SPEC INPUT FILE *** '
-    echo '*********************************************** '
-    echo ' '
-    set_trace
-    exit_code=3
-    DOSPC_WAV='NO'
-    DOBLL_WAV='NO'
+    export err=3
+    err_chk 'FATAL ERROR: NO TEMPLATE FOR SPEC INPUT FILE'
```

Comment:
```suggestion
    err_chk "FATAL ERROR: NO TEMPLATE FOR SPEC INPUT FILE"
```

Reply:
Single and double quotes will behave the same here.  Doubles should be used when a variable is being referenced in the echo.  That said, I'm OK with changing this.

Reply:
I figured we be consistent but noted. Thanks! :)